### PR TITLE
For Issue #84 @ Seed.sol:220 retrieveFundingTokens()

### PR DESCRIPTION
**Pull request by codebuster22**
For Issue #84  
Removed checked modifier as the admin can un-whitelist the user, after the user have funded it. This prevents user from retrieving funding amount.
added require() to check fundingAmount>0 so that even if the Seed is whitelisted or not, you can only retrieve only if you have funded it.

### CODE REVIEW

File: contracts/seed/Seed.sol [hunk 1]
```diff
@@ -217,9 +217,12 @@ contract Seed {
     /**
       * @dev         Returns funding tokens to user.
     */
-    function retrieveFundingTokens() public protected checked beforeMinimumReached {
+    function retrieveFundingTokens() public protected /* checked */ beforeMinimumReached {
```
Comment (kinspotting) [inline contracts/seed/Seed.sol:220]: pls remove the comment

File: contracts/seed/Seed.sol [hunk 2]
```diff
@@ -217,9 +217,12 @@ contract Seed {
...
         Lock storage tokenLock = tokenLocks[msg.sender];
         uint amount = tokenLock.fundingAmount;
+        // Used who have funded it can only retrieve the funding tokens back.
```
Comment (kinspotting) [inline contracts/seed/Seed.sol:223]: the comment is unclear

File: contracts/seed/Seed.sol [hunk 3]
```diff
@@ -217,9 +217,12 @@ contract Seed {
...
+        require(amount > 0, "Seed: zero funding amount");
+        // End
```
Comment (kinspotting) [inline contracts/seed/Seed.sol:225]: pls remove

File: contracts/seed/Seed.sol [hunk 4]
```diff
@@ -217,9 +217,12 @@ contract Seed {
     /**
       * @dev         Returns funding tokens to user.
     */
-    function retrieveFundingTokens() public protected checked beforeMinimumReached {
+    function retrieveFundingTokens() public protected /* checked */ beforeMinimumReached {
         Lock storage tokenLock = tokenLocks[msg.sender];
         uint amount = tokenLock.fundingAmount;
```
Comment (kinspotting) [inline contracts/seed/Seed.sol:222]: why do we create a whole new variable here?

Comment (kinspotting) [reply contracts/seed/Seed.sol:222]: it could be checked directly

**Review by kinspotting — CHANGES_REQUESTED** [review]

File: contracts/seed/Seed.sol (hunk 1 above)
Comment (codebuster22) [reply contracts/seed/Seed.sol:220]: Yes, waiting for tests to run successfully.

File: contracts/seed/Seed.sol (hunk 2 above)
Comment (codebuster22) [reply contracts/seed/Seed.sol]: Typo:- User*

**Review by kinspotting — APPROVED** [review]
lgtm

**Pull request merged**